src/global/Constants.h

```diff
@@ -189,12 +190,24 @@ static constexpr int DEFAULT_MAX_NUM_COLUMNS_STATIC_ID_TABLE = 5;
 inline auto& RuntimeParameters() {
   using ad_utility::detail::parameterShortNames::Double;
   using ad_utility::detail::parameterShortNames::SizeT;
+  using ad_utility::detail::parameterShortNames::StreamableParameter;
   // NOTE: It is important that the value of the static variable is created by
   // an immediately invoked lambda, otherwise we get really strange segfaults on
   // Clang 16 and 17.
   // TODO<joka921> Figure out whether this is a bug in Clang or whether we
   // clearly misunderstand something about static initialization.
   static ad_utility::Parameters params = []() {
+    auto ensureStrictPositivity = [](auto&& parameter) {
```

Comment:
Inside the lambda set `using namespace std::chrono_literals` and then use `0s` and `30s` as the literals.